public/less/_user.less

```diff
@@ -58,6 +58,18 @@
         .ui.repository.list {
             margin-top: 25px;
         }
+
```

Comment:
There was a CSS change that was just mereged into master so you'll likely need to recompile the LESS.

Reply:
I see, but as the change was in the `arc-green` theme where I didn't change anything, I don't need to regenerate css.

Reply:
excellent!